cypress/cypress/downloads/mentor.json

```diff
@@ -0,0 +1 @@
+undefined
```

Comment:
we need this as a file? maybe call it `mentor-undefined.json` if we really do this